src/TabComplete.js

```diff
@@ -45,21 +48,32 @@ class TabComplete {
         this.isFirstWord = false; // true if you tab-complete on the first word
         this.enterTabCompleteTimerId = null;
         this.inPassiveMode = false;
+        this.memberTabOrder = {};
+        this.memberOrderSeq = 0;
```

Comment:
What is this? A number to represent the member order? But how? Is it a max size? Missing explanatory comment.